docs/pages/getting-started/docker-compose.mdx

```diff
@@ -0,0 +1,217 @@
+description: How to get started with Teleport Open Source Edition using Docker Compose locally.
+h1: Get started with Docker Compose
+---
+
+This section will cover:
+
+- Getting started with a local Teleport cluster using docker compose.
+- Using Teleport with OpenSSH, ansible and Teleport's native client, `tsh`.
+
+## Prerequisites
+
+- Teleport v(=teleport.version=) Open Source or Enterprise.
+- Docker v(=docker.version=) or later and [`docker-compose`](https://docs.docker.com/compose/install/) v(=docker.compose.version=) or later.
+
+```bash
+docker-compose version
+# docker-compose version (=docker.compose.version=), build unknown
+
+docker version
+#Client: Docker Engine - Community
+# Version:           (=docker.version=)
+```
+
+## Step 1/3. Start teleport demo lab
+
+Let's use `docker-compose` to start Teleport demo lab - a configured local cluster:
+
+```bash
+# Download the quick start file from our Github repo
+curl -Lso teleport-lab.yml https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/docker/teleport-lab.yml
+# Start teleport demo lab using docker-compose
+docker-compose -f teleport-lab.yml up
+```
+
+<Admonition type="note" title="Note">
+You can later stop the Teleport lab using:
+
+```bash
+docker-compose -f teleport-quickstart.yml down
+```
+</Admonition>
+
+## Step 2/3. Explore teleport using CLI
+
+Let's jump into container with setup clients and explore Teleport:
+
+```bash
+docker exec -ti term /bin/bash
+```
+
+<Admonition type="note" title="Note">
+We will run all future commands from the `term` container.
+</Admonition>
+
+Welcome to Teleport Lab. With Teleport you can access servers, databases and web apps in your cluster.
+
+Let's Try a couple of commands to get started.
+Teleport speaks SSH. You can SSH into it using OpenSSH:
+
+```bash
+ssh root@luna.teleport
+```
+
+Teleport is a bastion server for your OpenSSH hosts. SSH into OpenSSH server and record all commands:
+
+```bash
+ssh root@mars.openssh.teleport
+```
+
+You can also run ansible on Teleport nodes and OpenSSH servers:
+
+```bash
+cd /etc/teleport.d/ansible && ansible all -m ping
+```
+
+Try Teleport's client command: `tsh`. It's like `ssh`, but with superpowers.
+Find all hosts matching label `env=example` and run `hostname` command:
+
+```bash
+tsh ssh root@env=example hostname
+```
+
+You can see Teleport's nodes registered in the cluster using `tsh ls` command:
+
+```bash
+tsh ls
+#Node Name     Address        Labels                     
+#------------- -------------- -------------------------- 
+#luna.teleport 127.0.0.1:3022 env=example, hostname=luna 
+```
+
+## Step 3/3. Explore teleport using web UI
+
+To create a user inside your Teleport container, use `docker exec`.
+
+This example command will create a Teleport user called `testuser` which is allowed to log in as either operating system user `root` or `ubuntu`:
+
+From terminal:
+
+```bash
+tctl users add testuser --roles=editor,access --logins=root,ubuntu,ec2-user
```

Comment:
Needs to be a full `docker exec teleport tctl users add testuser --roles=editor,access --logins=root,ubuntu,ec2-user`